frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -170,7 +169,10 @@ export const funnelLogic = kea<funnelLogicType>({
             breakdown_value,
         }),
         setStepReference: (stepReference: FunnelStepReference) => ({ stepReference }),
-        changeHistogramStep: (from_step: number, to_step: number) => ({ from_step, to_step }),
+        changeHistogramStep: (funnel_from_step?: number, funnel_to_step?: number) => ({
```

Comment:
Would be good to also remove the `histogram` characterization in this logic to something more general, maybe `changeStepRange` here for instance.